src/App/Fossa/Configuration.hs

```diff
@@ -115,7 +116,10 @@ readConfigFile file = do
 
 readConfigFileIO :: IO (Maybe ConfigFile)
 readConfigFileIO = do
-  config <- Diag.runDiagnostics $ runReadFSIO $ readConfigFile defaultFile
+  -- FIXME: we probably want to read from the target directory of analysis, not
```

Comment:
- [ ] TODO: Use target directory of analysis

Reply:
Yeah, looks like you've [ticketed this already](https://github.com/fossas/team-analysis/issues/692) 😬